workflows/cloudify_system_workflows/snapshots/postgres.py

```diff
@@ -205,25 +214,57 @@ def _get_status_reporters_update_query(self):
         query - updates the status reporters in the DB and
         protected_query - hides the credentials for the logs file
         """
-        base_query = """
-        INSERT INTO users (username, password, api_token_key)
+        create_user_query = """
+        INSERT INTO users (username, password, api_token_key, active, id)
         VALUES
            (
               '{0}',
               '{1}',
-              '{2}'
+              '{2}',
+              TRUE,
+              {3}
            )
         ON CONFLICT (username) DO
-        UPDATE SET password='{1}', api_token_key='{2}'
+        UPDATE SET password='{1}', api_token_key='{2}', active=TRUE, id={3}
```

Comment:
Dose it work? the `TRUE`? I think it is usually `t`

Reply:
It works 👍

Reply:
nice :)